sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -925,6 +925,13 @@ object SQLConf {
       .intConf
       .createWithDefault(10000)
 
+  val ARROW_EXECUTION_STREAM_ENABLE =
+    buildConf("spark.sql.execution.arrow.stream.enable")
+      .internal()
+      .doc("When using Apache Arrow, use Arrow stream protocol if possible.")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
is there any known problems? I think we should enable it by default, otherwise most users can't benefit from it